### DISCUSSION TITLE
Hope to enhance the SQL editor function

### DISCUSSION THREAD

**Discussion by z-hanzhe** [dbeaver/dbeaver#19381] [Q&A] [answered]
**The content is translated by Google, please forgive me if the description is wrong**

After selecting the database, pressing Ctrl + Enter will pop up the SQL editor, but this function is not very user-friendly

For example, when I want to modify the name of a SQL file, I find that there is no such function, and there is also no function to delete the SQL file.

And when I want to find the directory where the SQL file is located, I find that there is no function to directly open the directory where the file is located. I can only open the resource manager according to the prompts and find it slowly.

![image](https://user-images.githubusercontent.com/58802604/225185182-f5794e13-ddfa-47e0-bea7-aa408aa13ff5.png)

**Comment by ShadelessFox** [comment, marked as answer]
Hello @Java-Zhe,

All three actions can be achieved by using the project view:

![image](https://user-images.githubusercontent.com/35821147/225187878-be4c9c7a-1716-40b1-adcc-4c099fe20723.png)

- You can rename resources by pressing <kbd>F2</kbd>
- You can delete resources by pressing <kbd>Delete</kbd>
- You can open the containing directory in explorer by choosing <kbd>Show resource in explorer</kbd> from the context menu

Please let me know if you have any other questions left.
